drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -509,36 +514,38 @@ function DragAndDropTemplates(configuration) {
             item_bank_properties.attributes = {
                 'tabindex': 0,
                 'dropzone': 'move',
-                'aria-dropeffect': 'move',
-                'role': 'button'
+                'aria-dropeffect': 'move'
             };
         }
+        var main_element_properties = {
+            attributes: {
+                'role': 'group',
+                'aria-label': gettext('Drag and Drop Problem')
+            }
+        };
         return (
-            h('section.themed-xblock.xblock--drag-and-drop', [
+            h('div.themed-xblock.xblock--drag-and-drop', main_element_properties, [
                 problemTitle,
                 problemProgress,
-                h('section.problem', [
+                h('div.problem', [
                     problemHeader,
                     h('p', {innerHTML: ctx.problem_html}),
                 ]),
-                h('section.drag-container', {}, [
+                h('div.drag-container', {}, [
                     h('div.item-bank', item_bank_properties, [
                         h('p', { className: 'zone-description sr' }, gettext('Item Bank')),
                         renderCollection(itemTemplate, items_in_bank, ctx),
```

Comment:
Does it make more sense to put the `Item Bank` description in as an `aria-label` on the `div.item-bank`, instead of an sr-visible `p` tag?  Could use `role="group"` on that div too.

Then the `div.item-bank` would also match the `div.target` below.